econet_hvac.yaml

```diff
@@ -46,12 +46,16 @@ sensor:
     sensor_datapoint: F_ACTCFM
     state_class: "measurement"
     entity_category: "diagnostic"
+    filters:
+      - delta: 10.0
```

Comment:
Should this actually be 9.9 to match the way you're using delta for other sensors?